mdm/orientdb-wrapper/src/main/java/com/gentics/mesh/core/data/dao/impl/GroupDaoWrapperImpl.java

```diff
@@ -84,6 +86,17 @@ public Result<? extends HibRole> getRoles(HibGroup group) {
 		return groupRoot.getRoles(graphGroup);
 	}
 
+	@Override
+	public Map<HibGroup, Collection<? extends HibRole>> getRoles(Collection<HibGroup> groups) {
```

Comment:
This could've been put into PersistingGroupDao as default method, since its does not utilize any DB specific data.

Reply:
Yes but we don't want that as default implementation, because it's inefficient for Mesh SQL.

Reply:
SQL overrides it anyway.